csm/conf/csm_cleanup.py

```diff
@@ -79,38 +86,73 @@ def remove_old_indexes(es, arg_d, arg_n, arg_e):
         else:
             Log.debug("Nothing to remove")
 
-def process_stats(args):
-    # Pass arguments to worker function
-    es = esCleanup(const.CSM_CLEANUP_LOG_FILE, const.CSM_LOG_PATH)
-    remove_old_indexes(es, args.d, args.n, args.e)
+def clean_indexes(es, no_of_days, host_port):
+    for index,field in index_field_map.items():
+        Log.debug(f"Removing data for old index:{index} for {no_of_days} days.")
+        es.remove_old_data_from_indexes(no_of_days, host_port, [index], field)
+    remove_old_indexes(es, no_of_days, host_port, args.emulate)
+
+def get_du_data():
+    try:
+        cmd = "sudo du -BM /var/log/elasticsearch/"
+        sp_es = SimpleProcess(cmd)
+        Log.debug(f"Running {cmd}")
+        res = sp_es.run()
+        Log.debug(f"Resulted: {res}")
+        res = int(res[0].decode("utf-8").split('\t')[0].split('M')[0])
+        return res
+    except Exception as e:
+        Log.error(f"Error in processing {cmd}: {e}")
+        raise Exception(f"Error in processing {cmd}: {e}")
```

Comment:
@ajaysrivas , @kupranay Is raising exception correct or we just log the exception?